tests/tests.js

```diff
@@ -4,42 +4,137 @@ const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
 
-describe('calculateProductPrice', () => {
-  it('returns the price for a voluntary life product for a single employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee'],
-      coverageLevel: [{ role: 'ee', coverage: 125000 }],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
-
-    expect(price).to.equal(39.37)
+describe('Pricing', () => {
+
+  describe('formatPrice', () => {
+    it('returns a price with two decimal places not rounded when given a number with multiple decimal places', () => {
+      const formattedprice = pricing.formatPrice(15.1566663)
+      expect(formattedprice).to.equal(15.15)
+    })
+  })
+
+  describe('calculateLTDPrice', () => {
+    it('returns the price of an LTD plan for a single employee without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [{ role: 'ee', coverage: 125000 }],
+      }
+      const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+    })
+
+    it('returns the price for a disability product for an employee without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee']
+      }
+      const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+    })
+
+  })
```

Comment:
These tests are testing the same thing. LTD calculates off of the employee salary and does not use coverage levels at all. That first test can just be deleted.